airbyte-integrations/connectors/source-freshdesk/sample_files/configured_catalog.json

```diff
@@ -548,11 +526,11 @@
             }
           }
         },
-        "supported_sync_modes": ["full_refresh"],
-        "source_defined_cursor": false,
+        "supported_sync_modes": ["full_refresh", "incremental"],
```

Comment:
does this source support full refresh if the sync mode = full refresh? or does it always look at state ?

Reply:
unfortunately not, I have an idea to force this in the base source, but I think it makes sense to do this in a separate PR